src/libraries/MathLib.sol

```diff
@@ -8,33 +8,33 @@ uint256 constant WAD = 1e18;
 /// @custom:contact security@morpho.org
 /// @notice Library to manage fixed-point arithmetic.
 library MathLib {
-    /// @dev (x * y) / WAD rounded down.
```

Comment:
Can you remove unnecessary comments in the constructor and in IIrm please ?

Reply:
Is it related to the line of your comment or not at all?

Reply:
Ah no it is not related to this line. I did not know where to put it, I'll explain that it is not related next time

Reply:
note that this exists 
<img width="527" alt="image" src="https://github.com/morpho-org/morpho-blue/assets/74971347/dce1a8bc-5649-4917-9a5d-6047cb6097d6">